config/kubernetes/org.yaml

```diff
@@ -1351,8 +1351,8 @@ teams:
     - cici37
     - jpbetz
     - jprzychodzen
+    - mmamczur
```

Comment:
It would be better to split this into its own folder like what the other cloud providers did:

https://github.com/kubernetes/org/tree/main/config/kubernetes/provider-azure

Reply:
We don't have a good way to set OWNERS files on this entire file.

Reply:
That would be great. Once you split off to a subdirectory, you wouldn't need org admins to approve PRs.

Reply:
I changed the PR. I'm not sure if it done correctly so please take a look if anything is missing

Reply:
@palnabarun would you be able to approve this change? I'm not sure who I can ask to merge this